### ISSUE TITLE
Open Directory Opens ~ Instead of Selected Directory on Linux

### ISSUE THREAD

**Issue by sj4nz** [atom/atom#4848]
* Atom version: v0.165.0
* OS: elementaryOS 0.2 "Luna" (64-bit)

Minor, but annoying if you like your projects hoisted into the sidebar. This seems to work fine on Windows 7.

**Comment by sj4nz** [comment]
Workaround: Navigate to desired directory in sidebar and "Open in new window."

So "Control-Shift-O" doesn't work, but "Open in new window" works regarding directories.

**Comment by DarkArc** [comment]
I can't reproduce this on 0.180.0

**Comment by DarkArc** [comment]
This also seems to be a duplicate of issue #3063.

**Comment by sj4nz** [comment]
Still true for me with Atom v0.179.0 and elementaryOS Version: 0.2.1 "Luna" ( 64-bit ).

**Comment by 50Wliu** [comment]
Closing this in favor of #3063 (which in turn is waiting on atom/atom-shell#613)-thanks @DarkArc.
